packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -54,6 +54,7 @@ describe("Garbage Collection Tests", () => {
         updateStateBeforeGC: async () => {},
         getGCData,
         updateUsedRoutes,
+        closeFn: () => {},
```

Comment:
should this set `closeCalled` to `true`?

Reply:
Yes, it should, otherwise the test will fail

Reply:
Yes, you are right.